.gitlab/deploy_7/docker.yml

```diff
@@ -237,12 +249,18 @@ deploy_google_container_registry-dogstatsd:
       --image ${REPOSITORY}-amd64:${VERSION}-win1809,windows/amd64
       --image ${REPOSITORY}-amd64:${VERSION}-win1909,windows/amd64
       --image ${REPOSITORY}-amd64:${VERSION}-win2004,windows/amd64
+      --image ${REPOSITORY}-amd64:${VERSION}-win1809-core,windows/amd64
```

Comment:
You cannot add to existing tag, otherwise the manifest will have multiple images for the same Windows version, and one of them won't be accessible.